tests/Jellyfin.Networking.Tests/NetworkParseTests.cs

```diff
@@ -299,15 +299,38 @@ public void HasRemoteAccess_GivenWhitelist_AllowsOnlyIPsInWhitelist(string addre
             var startupConf = new Mock<IConfiguration>();
             using var nm = new NetworkManager(NetworkParseTests.GetMockConfig(conf), startupConf.Object, new NullLogger<NetworkManager>());
 
-            Assert.NotEqual(nm.HasRemoteAccess(IPAddress.Parse(remoteIP)), denied);
+            Assert.Equal(nm.ShouldAllowServerAccess(IPAddress.Parse(remoteIP)), expectedResult);
```

Comment:
First argument is the expected result
```suggestion
            Assert.Equal(expectedResult, nm.ShouldAllowServerAccess(IPAddress.Parse(remoteIP)));
```